tools/sim/README.md

```diff
@@ -25,3 +25,17 @@ You can control openpilot driving in the simulation with the following keys
 |   3   |   Cruise cancel   |
 |   q   |     Exit all      |
 
+## Arguments
```

Comment:
This should be done with argparse.

Reply:
That is the argparse output. Should I add all of it?
If so, I would just change `bridge.py` to `start_openpilot_docker.sh`, because that's the script users execute. 
```
usage: bridge.py [-h] [--joystick] [--town TOWN] [--spawn_point NUM_SELECTED_SPAWN_POINT] [--cloudyness CLOUDYNESS] [--precipitation PRECIPITATION] [--precipitation_deposits PRECIPITATION_DEPOSITS] [--wind_intensity WIND_INTENSITY] [--sun_azimuth_angle SUN_AZIMUTH_ANGLE]
                 [--sun_altitude_angle SUN_ALTITUDE_ANGLE]

Bridge between CARLA and openpilot.

optional arguments:
  -h, --help            show this help message and exit
  --joystick
  --town TOWN
  --spawn_point NUM_SELECTED_SPAWN_POINT
  --cloudyness CLOUDYNESS
  --precipitation PRECIPITATION
  --precipitation_deposits PRECIPITATION_DEPOSITS
  --wind_intensity WIND_INTENSITY
  --sun_azimuth_angle SUN_AZIMUTH_ANGLE
  --sun_altitude_angle SUN_ALTITUDE_ANGLE
```

Reply:
Instead of pasting the output in the README, just give the help command.

Reply:
Updated the README